ex02.ex

```diff
@@ -30,21 +30,25 @@ defmodule Ex02 do
   # numbers, and second should be the difference                         #
   ########################################################################
 
-  list2a = your_anonymous_function(1, 2)
+  list2a = fn a, b -> [a+b, a-b] end
 
   assert list2a.(1, 2)    == [ 3, -1 ]
   assert list2a.(-1, 100) == [ 99, -101 ]
+  assert list2a.(-1, -1) == [-2, 0]
+  assert list2a.(-0.1, 0.1) == [0, -0.2]
 
   ##################
   # 2.2:  5 points #
   ##################################
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([&1 + &2, &1 - &2])
```

Comment:
Good question... I imagine it was because I was using so many symbols and it looked confusing to me without them.  I'll keep that in mind.